openlibrary/templates/covers/book_cover.html

```diff
@@ -14,7 +14,7 @@
 $if size == "M":
     <div class="coverMagic">
             <div class="SRPCover bookCover" style="display: $cond(cover_url, 'block', 'none');">
-                <a href="$cover_lg" class="coverLook" title="Pull up a bigger book cover"><img itemprop="image" src="$cover_url" class="cover" alt="Cover of: $title $_('by') $author_names"/></a>
+                <a href="$cover_lg" class="coverLook" title="Pull up a bigger book cover"><img itemprop="image" src="$cover_url" class="cover" alt="Cover of: $title $_(' | ') $author_names"/></a>
```

Comment:
this doesn't need the underscore now, there is no reason to pass the character to python if you just want the character.

`alt="Cover of: $title | $author_names"`
will do the same thing